src/Service.Tests/TestHelper.cs

```diff
@@ -115,10 +115,7 @@ public static RuntimeConfig AddMissingEntitiesToConfig(RuntimeConfig config, str
         public const string SAMPLE_SCHEMA_DATA_SOURCE = SCHEMA_PROPERTY + "," + @"
             ""data-source"": {
               ""database-type"": ""mssql"",
-              ""connection-string"": ""testconnectionstring"",
-              ""options"": {
-                ""set-session-context"": true
-                }
```

Comment:
removing options here to cover non-MsSql databases where these options will be null to demonstrate that null dereference no longer occurs?

Reply:
to check for null dereference, modified the tests that start the engine and actually execute some queries. Removing here, to simulate deserialization results in options being null.